modules/apps/object/object-service/bnd.bnd

```diff
@@ -2,6 +2,6 @@ Bundle-Name: Liferay Object Service
 Bundle-SymbolicName: com.liferay.object.service
 Bundle-Version: 1.0.201
 Liferay-Client-Extension-Batch: com/liferay/object/internal/batch
-Liferay-Require-SchemaVersion: 9.2.2
+Liferay-Require-SchemaVersion: 9.2.3
```

Comment:
@p-albuquerque if you are changing a column name, then it's a major change. Please see this [article](https://liferay.atlassian.net/wiki/spaces/ENGCOREUPGRADE/pages/1654719711/Upgrade+Development+Training#Major-changes).